math/mean/mean.go

```diff
@@ -0,0 +1,27 @@
+package mean
+
+import (
+	"fmt"
+)
+
+func mean(values []float64) float64{
```

Comment:
Please also consider using generics here, this is clearly something that would benefit from generics.

Reply:
Just added the generics. Please review.